Src/WitsmlExplorer.Frontend/components/Modals/LogCurvePriorityModal.tsx

```diff
@@ -107,7 +131,6 @@ export const LogCurvePriorityModal = (
                 value={newCurve}
               />
               <Button
-                variant="contained_icon"
```

Comment:
Was it intentional to remove this?

Reply:
yes since I have to use button from mui/material for file upload button